jgrapht-core/src/main/java/org/jgrapht/alg/isomorphism/ColorRefinementIsomorphismInspector.java

```diff
@@ -99,6 +104,55 @@ public ColorRefinementIsomorphismInspector(Graph<V, E> graph1, Graph<V, E> graph
         this.isForest = false;
     }
 
+    /**
+     * Returns whether <code>isomorphism</code> is an isomorphism between <code>graph1</code> and <code>graph2</code>
+     *
+     * @param graph1 the first graph
+     * @param graph2 the second graph
+     * @param isomorphism the isomorphim to test
+     * @return whether <code>isomorphism</code> is an isomorphism between <code>graph1</code> and <code>graph2</code>
+     */
+    public boolean assertIsomorphism(Graph<V, E> graph1, Graph<V, E> graph2, IsomorphicGraphMapping<V, E> isomorphism) {
```

Comment:
This only verifies the forward direction of the mapping; it needs to check both forward and reverse.

I forgot that we already have a complete implementation in IsomorphismTestUtil.areIsomorphic.  It's currently used by tests only, but it would make sense to move it to the IsomorphicGraphMapping class itself, renaming it to isValid.  It doesn't really need to take the graph1/graph2 parameters since IsomorphicGraphmapping already has those as data members.